xml/System.IO.Compression/GZipStream.xml

```diff
@@ -392,7 +392,7 @@
         <param name="cback">An optional asynchronous callback, to be called when the read operation is complete.</param>
         <param name="asyncState">A user-provided object that distinguishes this particular asynchronous read request from other requests.</param>
         <param name="state">A user-provided object that distinguishes this particular asynchronous read request from other requests.</param>
-        <summary>Begins an asynchronous read operation. (Consider using the <see cref="M:System.IO.Stream.ReadAsync(System.Byte[],System.Int32,System.Int32)" /> method instead.)</summary>
+        <summary>Begins an asynchronous read operation. (Consider using the <see cref="M:System.IO.Compression.GZipStream.ReadAsync(System.Byte[],System.Int32,System.Int32)" /> method instead.)</summary>
```

Comment:
ok, you got a couple of build warnings. This change is actually bad since the method on GZipStream is inherited from Stream, so you don't have API in the GZipStream class itself

Reply:
Oh you're right, I can see that now, thanks for letting me know.

There are only two options in GZipStream:

```
ReadAsync(byte[] array, int offset, int count, CancellationToken cancellationToken);
ReadAsync(Memory<byte> buffer, CancellationToken cancellationToken = default(CancellationToken));
```

But there isn't one with (byte[], int, int).

I'll amend the code.